homeworks/denys.viitenko_danviitenko/homework_3/RememberMe_2/RememberMe_2/ViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ViewController.swift
+//  RememberMe_2
+//
+//  Created by Dan on 11/25/18.
+//  Copyright © 2018 Dan Viitenko. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    let flipCountLabel: UILabel! = UILabel()
```

Comment:
Нет необходимости указывать force unwrap для лейбы, в ситуации, когда мы сразу же ее инициализируем.

Рекомендую повторить:
https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html#ID330

```suggestion
    let flipCountLabel: UILabel = UILabel()
```